### PR TITLE
(test) O3-1885: Implement edit patient e2e test

### PR DESCRIPTION
**Pull request by RandilaP**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
This pull request adds an end-to-end test for editing patient details. The test covers the following scenarios:

- Editing the patient's first name
- Editing the patient's middle name
- Editing the patient's last name
- Editing the patient's gender
- Editing the patient's date of birth
- Editing the patient's address
- Editing the patient's phone number

## Screenshots

*None.*

## Related Issue
https://issues.openmrs.org/browse/O3-1885

## Other

*None.*

### CODE REVIEW

**Comment by RandilaP** [comment]
@jayasanka-sack Could you please review this PR

File: e2e/pages/registrationAndEditPage.ts [hunk 1]
```diff
@@ -36,6 +36,8 @@ export class PatientRegistrationPage {
   readonly emailInput = () => this.page.locator('#email');
   readonly createPatientButton = () => this.page.locator('button[type=submit]');
 
+  readonly updateButton = () => this.page.locator('button:');
+
```
Comment (jayasanka-sack) [inline e2e/pages/registrationAndEditPage.ts]: Remove this unused variable

File: e2e/pages/registrationAndEditPage.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ export interface PatientRegistrationFormValues {
   email?: string;
 }
 
-export class PatientRegistrationPage {
+export class registrationAndEditPage {
```
Comment (jayasanka-sack) [inline e2e/pages/registrationAndEditPage.ts:22]:
```suggestion
export class RegistrationAndEditPage {
```

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Thanks @RandilaP ! I added some comments. Let me know if you require further clarification. :)

**Review by anjula-sack — APPROVED** [review]
Good job! @RandilaP LGTM

**Review by jayasanka-sack — APPROVED** [review]
Thanks @RandilaP !

**Pull request merged**